spec/unit/remove_task_spec.rb

```diff
@@ -40,11 +41,10 @@
 
     it "shows also a success message if verbose is true" do
       create_machinery_dir
-      expect(Machinery::Ui).to receive(:puts).with(
+      remove_task.remove(store, test_name, { :verbose => true })
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.